$OpenBSD: patch-suricata_yaml_in,v 1.4 2019/03/26 13:46:24 bluhm Exp $

Switch user and group to avoid running as root.
To remove pid file its directory must be writable by suricata user.
Remove rules files not present by default.

Index: suricata.yaml.in
--- suricata.yaml.in.orig
+++ suricata.yaml.in
@@ -1042,9 +1042,9 @@ asn1-max-frames: 256
 ##
 
 # Run suricata as user and group.
-#run-as:
-#  user: suri
-#  group: suri
+run-as:
+  user: _suricata
+  group: _suricata
 
 # Some logging module will use that name in event as identifier. The default
 # value is the hostname
@@ -1053,7 +1053,7 @@ asn1-max-frames: 256
 # Default location of the pid file. The pid file is only used in
 # daemon mode (start Suricata with -D). If not running in daemon mode
 # the --pidfile command line option must be used to create a pid file.
-#pid-file: @e_rundir@suricata.pid
+pid-file: @e_rundir@suricata.pid
 
 # Daemon working directory
 # Suricata will change directory to this one if provided
@@ -1848,9 +1848,21 @@ mpipe:
 ## file configuration".
 ##
 
-@no_suricata_update_comment@default-rule-path: @e_defaultruledir@
-@no_suricata_update_comment@rule-files:
-@no_suricata_update_comment@ - suricata.rules
+default-rule-path: @e_sysconfdir@rules
+rule-files:
+
+  - app-layer-events.rules
+  - decoder-events.rules
+  #- dnp3-events.rules
+  - dns-events.rules
+  - files.rules
+  - http-events.rules
+  #- modbus-events.rules
+  #- nfs-events.rules
+  #- ntp-events.rules
+  - smtp-events.rules
+  - stream-events.rules
+  - tls-events.rules
 
 ##
 ## Advanced rule file configuration.
